app/views/items/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="col-md-4">
+  <div class="form-group">
+    <%= form_for [@user, @user.items.build], remote: true do |f| %>
+      <%= f.label :Item %>
```

Comment:
symbols really shouldn't begin with a capitalized letter.  Additionally, labels should match their respective inputs.